pkg/crd/crd.go

```diff
@@ -18,9 +18,12 @@ import (
 )
 
 type Client struct {
-	rest           *rest.RESTClient
-	BindingWatcher cache.SharedInformer
-	IdWatcher      cache.SharedInformer
+	rest                *rest.RESTClient
+	BindingListWatch    *cache.ListWatch
+	BindingWatcher      cache.SharedInformer
+	IDListWatch         *cache.ListWatch
+	IdWatcher           cache.SharedInformer
+	AssignedIDListWatch *cache.ListWatch
```

Comment:
The naming of `..Watch` gives the impression that they are all the same type which the are not. Can we use a different naming for informers vs watchers?

Reply:
Thank you @khenidak.